recipes/libelf/all/conanfile.py

```diff
@@ -86,13 +87,27 @@ def _package_autotools(self):
         if self.settings.os == "Linux" and self.options.shared:
             os.remove(os.path.join(self.package_folder, "lib", "libelf.a"))
 
+    @property
+    def _user_info_build(self):
+        # If using the experimental feature with different context for host and
+        # build, the 'user_info' attributes of the 'build_requires' packages
+        # will be located into the 'user_info_build' object. In other cases they
+        # will be located into the 'deps_user_info' object.
+        return getattr(self, "user_info_build", None) or self.deps_user_info
```

Comment:
https://docs.conan.io/en/latest/changelog.html?highlight=user_info_build#id257

Hmm

Reply:
@jgsogo could you check, if it's correct way?

Reply:
Yes, it is sound. Information from the _build_ context can be retrieved using `self.user_info_build`. 👍